console/console-init/ui/src/modules/iot-device/containers/EditCredentialsContainer/edit-credentials.module.css

```diff
@@ -0,0 +1,3 @@
+.marginLeft {
+  margin-left: 25px;
```

Comment:
I am not professional in UI design but this concrete margins never work :D What if user will have 4k ultrawide monitor? I would go with some more flexible units like rem or % for example.